apps/native-component-list/src/screens/Image/ImageTestScreen.tsx

```diff
@@ -106,6 +115,18 @@ export default function ImageTestScreen({ navigation, route }: Props) {
     setLoadDemanded(true);
   };
 
+  const onPressMethod = async () => {
+    const uri = test.props.source.uri;
+    if (test.method != null) {
```

Comment:
I believe, if property is not set, it has value of `undefined`, therefore we should rather check against `undefined`:
 
```suggestion
    if (test.method !== undefined) {
```
or maybe just against any falsy value

```suggestion
    if (test.method) {
```

This situation repeats in few places.